src/meshapi/views/forms.py

```diff
@@ -195,7 +195,7 @@ def join_form(request):
     )
 
 
-def get_next_free_nn() -> int:
+def get_next_free_nn_from_old_installs() -> int:
     defined_nns = set(
         Install.objects.exclude(install_status=Install.InstallStatus.REQUEST_RECEIVED, network_number__isnull=True)
```

Comment:
Just checking: Should we be excluding `REQUEST_RECEIVED`? Do we know for sure at this point in the code that no `REQUEST_RECEIVED` Installs have NNs?